.github/tests/application/.env.example

```
@@ -1,17 +1,17 @@
-WORDPRESS_ARG_THEME_ZIP_URL=https://downloads.wordpress.org/theme/twentytwelve.4.6.zip
+WORDPRESS_ARG_BASE_IMAGE=wordpress:6.9-php8.5-apache
+WORDPRESS_ARG_PLUGIN_BBPRESS_ZIP_URL=https://downloads.wordpress.org/plugin/bbpress.2.6.14.zip
+WORDPRESS_ARG_THEME_ZIP_URL=https://downloads.wordpress.org/theme/twentytwelve.4.7.zip
 
 CLI_ARG_BASE_IMAGE=wordpress:cli
 
-MYSQL_DATABASE=
-MYSQL_USER=
+MYSQL_DATABASE=wp
+MYSQL_USER=wp
 MYSQL_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MYSQL_PASSWORD key should go before the MYSQL_USER key

(UnorderedKey)